v0-991/learn/how-to-deploy-and-run-ballerina-programs.md

```diff
@@ -216,6 +216,7 @@ Add the following code to the `hello_world_docker.bal` file.
 ```ballerina
 import ballerina/http;  
 import ballerinax/docker;  
+import ballerina/log;

```

Comment:
Why do we need to import log here? Seems it is not used anywhere.

Reply:
Removed "import/log" from v0-991.